test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -732,6 +732,38 @@
       initPods: 2000
       measurePods: 5000
 
+# Measure throughput of regular schedulable pods that are interleaved by unschedulable pods injected at 100/s rate.
```

Comment:
Can you also add a comment in the base "Unschedulable" test?

How do they compare in terms of throughput? Maybe we don't need to keep the original one?

Reply:
Right, the original test is a bit unclear to me because of using skipWaitToCompletion: true. Initially I thought that the it's executed sequentially, but it's very likely the idea behind it was to test the same thing. I can replace it.